packages/vite/src/executors/test/vitest.impl.ts

```diff
@@ -52,6 +52,9 @@ export default async function* runExecutor(
     ...options,
     root: projectRoot,
     reporters: [...(options.reporters ?? []), 'default', nxReporter],
+    coverage: {
+      reportsDirectory: options.reportsDirectory,
```

Comment:
Does it matter if this is undefined? I assume it'll use what's in vite config in that case?

Reply:
Hmmmm I would think it does not matter, yes, and it picks up whatever is in `vite.config.ts`. At least that's how it works in the other cases.